src/ert/_c_wrappers/enkf/subst_config.py

```diff
@@ -93,8 +93,14 @@ def __init__(self, config_content=None, config_dict=None):
                 "The name of a file with a list of run directories.",
             )
 
+            if ConfigKeys.NUM_CPU in config_dict:
+                subst_list.addItem(
```

Comment:
I think this will add the item twice when `os.path.isfile(file_path) and os.access(file_path, os.R_OK)` (line 108). That might become problematic down the line.

Reply:
i think the logic is

- if the NUM_CPU is given in the config dict, use that
- if it's not given in the config dict, try to find it in the DATA FILE

so i'm not sure i follow how it will be added twice?